optimum/habana/diffusers/pipelines/pipeline_utils.py

```diff
@@ -158,15 +158,6 @@ def __init__(
                     f"`gaudi_config` must be a string or a GaudiConfig object but is {type(gaudi_config)}."
                 )
 
-            if self.gaudi_config.use_torch_autocast:
-                if bf16_full_eval:
-                    logger.warning(
-                        "`use_torch_autocast` is True in the given Gaudi configuration but "
-                        "`torch_dtype=torch.bfloat16` was given. Disabling mixed precision and continuing in bf16 only."
-                    )
-                    self.gaudi_config.use_torch_autocast = False
```

Comment:
Why removing this? I don't see how it impacts the issue you want to solve.

Reply:
answer as below